rules_conditions.md

```diff
@@ -201,6 +201,15 @@ For most cases, use `not-exists` in a rule with a scope of "span".
 WARNING: Rules can have `Scope: trace` or `Scope: span`; `not-exists` used with `Scope: trace` will be true if **any** single span in the entire trace matches the negative condition.
 This is almost never desired behavior.
 
+### has-root-span
+
+Tests if the trace as a whole has a root span.
+
+The `Value` parameter can either be `true` or `false`.
+
+WARNING: `has-root-span` does not check if a given span is a root span.
+It only checks if a trace has a root span.
```

Comment:
```suggestion
NOTE: `has-root-span` does not check if a given span **is** a root span,
it checks if the containing trace **has** a root span.
```

Reply:
Huh, my suggestion includes the opposite of Yingrong's -- I feel like "WARNING" is too strident and want this to be a note.